guides/session.rst

```diff
@@ -183,10 +183,17 @@ Resetting the Session
 ---------------------
 
 The primary aim for Mink is to provide a single consistent web browsing API
-for acceptance tests. But a very important part in testing is isolation.
+for acceptance tests. But a very important part in testing is isolation.  
+Mink tries to automatically "soft" isolate tests from each other without
```

Comment:
Mink does not do anything automatically. Both ``reset`` and ``stop`` must be called explicitly